lib/client.js

```diff
@@ -112,16 +112,16 @@ Client.prototype.send = function() {
   // Respect IRC line length limit of 512 chars
   // https://tools.ietf.org/html/rfc2812#section-2.3
   if (message.length > 512) {
-    var parts   = message.split(':');
```

Comment:
This is for messages sent from the server to the client. All the messages sent from the server to the client have `' :'` in them, because it's part of the irc protocol.

```
var prefixes = ':source PRIVMSG target :';
```

And while it's legal to skip the : if the message doesn't have any spaces, there's no good reason to do that and this server doesn't do it.